pkg/plugin/v2/api.go

```diff
@@ -148,7 +148,7 @@ func (p *createAPIPlugin) Validate() error {
 		}
 
 		// Check that the provided group can be added to the project
-		if p.config.IsV2() && !p.config.MultiGroup &&
+		if (p.config.IsV2() || p.config.IsV3()) && !p.config.MultiGroup &&
```

Comment:
```suggestion
		if ( !p.config.IsV1()&& !p.config.MultiGroup &&
```

Reply:
I think the intent here is to imply that v2 and v3 have multigroups. Your change is shorter but I'd rather leave this as-is for now.